### PR TITLE
Adding HackerNews posts #62

### PR DESCRIPTION
**Pull request by jbgerth**
I added HackerNews as a new command. It lets you post the top x article references. I also fixed the help command, that got broken during the latest upgrade, and removed the `.husky` files that did seem to work. 

The structure of the command is the first version to abstract the Discord API. I'll add a CLI interface for testing, but feel free to take a look already 😉

### CODE REVIEW

**Comment by jbgerth** [comment]
Just realized, that I named the branch wrong 😅

File: src/commands/cereal/hackernews.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  }
+  return numberOfPosts;
+};
+
+/**
+ * Get the news from Hackernews and return a funktion to create a properly formatted output
+ * @param url URL of hackernews
+ * @param args Arguments of the command
+ * @param createPosts A Funktion to create a post
+ * @returns A Funktion that creates a post
```
Comment (coinzz) [inline src/commands/cereal/hackernews.ts:83]: A function

File: src/commands/cereal/hackernews.ts [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+/**
+ * Parse args. The args can only be integers and 0 < x < 11
+ * @param args arguments of the command
+ * @returns Parsed args to get the number of posts to create
+ */
+const parseArgs = (args: string[]): number => {
+  let numberOfPosts = 5;
+
+  if (args) {
```
Comment (coinzz) [inline src/commands/cereal/hackernews.ts:60]: Is it correct that numberOfPosts shall be 5 if there are no args?

File: src/commands/cereal/hackernews.ts [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    if (args.length === 1) {
+      const parsedArg = parseInt(args[0]);
+      if (_.isInteger(parsedArg)) {
+        numberOfPosts = Math.abs(parsedArg);
+        if (numberOfPosts >= 10) {
+          numberOfPosts = 10;
+        }
```
Comment (coinzz) [inline src/commands/cereal/hackernews.ts:67]: Could be rewritten:
numberOfPosts = Math.max(parsedArg, 0);
numberOfPosts = Math.min(parsedArg, 11);

**Review by coinzz — APPROVED** [review]
Looking good so far :)

**Review by Xol — APPROVED** [review]
Looks good to me! 👍🏻

**Pull request merged**